core/templates/dev/head/pages/subtopic-viewer-page/subtopic-viewer-page.controller.ts

```diff
@@ -63,11 +63,11 @@ angular.module('oppia').directive('subtopicViewerPage', [
           $rootScope.loadingMessage = 'Loading';
           SubtopicViewerBackendApiService.fetchSubtopicData(
             ctrl.topicName, ctrl.subtopicId).then(
-            function(subtopicDataDict) {
+            function(subtopicDataObject) {
               ctrl.pageContents =
                 SubtitledHtmlObjectFactory.createFromBackendDict(
-                  subtopicDataDict.page_contents.subtitled_html);
-              ctrl.subtopicTitle = subtopicDataDict.subtopic_title;
+                  subtopicDataObject.getPageContents().subtitled_html);
```

Comment:
Just FYI, the `createFromBackendDict` here wouldn't be needed here if the above change is done. Just `subtopicDataObject.getPageContents().getSubtitledHtml()` would be enough.